client/src/components/screens/PatientVariant/components/VariantNavigation.jsx

```diff
@@ -154,6 +161,29 @@ class VariantNavigation extends React.Component {
     ));
   }
 
+  handleMenuSelection(e) {
+    let { activeMenu } = this.state;
+    activeMenu = activeMenu[0] === e.key ? [] : [e.key];
+    this.setState({
+      activeMenu,
+    });
+  }
+
+  handleClickOutside(event) {
+    const openMenu = document.querySelector('.submenuOpen');
+    const clickX = event.clientX;
+    const clickY = event.clientY;
+    if (openMenu) {
+      const menuX = Number(openMenu.style.left.replace('px', ''));
+      const menuY = Number(openMenu.style.top.replace('px', ''));
+      if (clickX < menuX || clickX > (menuX + 300) || clickY < menuY || clickY > menuY + openMenu.offsetHeight) {
```

Comment:
Where does the 300 come from? Can it be obtained from a function call or a constant?